### ISSUE TITLE
Enum34 causing problems in Python 3.4+

### ISSUE THREAD

**Issue by bgyori** [ndexbio/ndex2-client#76]
Currently, `enum34` is in the required installs list in setup.py and requirements.txt (see https://github.com/ndexbio/ndex2-client/issues/8). This now tends to cause problems in higher versions of Python and `enum34` needs to be manually uninstalled,  e.g.,
```
    File "/sw/miniconda/lib/python3.7/locale.py", line 16, in <module>
      import re
    File "/sw/miniconda/lib/python3.7/re.py", line 143, in <module>
      class RegexFlag(enum.IntFlag):
  AttributeError: module 'enum' has no attribute 'IntFlag'
```

Since `enum` and `enum34` are only needed for Python <= 3.3, which are pretty outdated, would it make sense at this point to remove them from the setup?

**Comment by coleslaw481** [comment]
Good point. I'll take those out for the `4.0.0` release

**Comment by ariutta** [comment]
Another option could be to update requirements.txt something like this:

```
enum34; python_version < "3.4"
```

**Comment by coleslaw481** [comment]
Oh, that is a good idea, let me test that out. thanks

**Comment by coleslaw481** [comment]
Added suggested change. Fixed with 3.3.2 release. Thanks again for the fix.
